.travis.yml

```diff
@@ -34,13 +34,11 @@ language: python
 
 matrix:
   include:
-    - python: 2.6
-      env: NUMPY=numpy==1.6 MOCK=mock
     - python: 2.7
-      env: MOCK=mock PANDAS=pandas
-    - python: 3.3
+      env: MOCK=mock NUMPY=numpy==1.6
```

Comment:
Are there any plans to drop support for numpy 1.6? I think version 1.7 came out in early 2013, and supporting 1.6 seems to accumulate some preprocessor cruft.